repos/fdbt-site/tests/pages/api/returnService.test.ts

```diff
@@ -0,0 +1,96 @@
+import returnService from '../../../src/pages/api/returnService';
+import {
+    expectedReturnTicketWithAdditionalService,
+    getMockRequestAndResponse,
+    mockRawService,
+} from '../../testData/mockData';
+import * as auroradb from '../../../src/data/auroradb';
+import {
+    MATCHING_JSON_ATTRIBUTE,
+    MATCHING_JSON_META_DATA_ATTRIBUTE,
+    TXC_SOURCE_ATTRIBUTE,
+} from '../../../src/constants/attributes';
+import * as userData from '../../../src/utils/apiUtils/userData';
+
+beforeEach(() => {
+    jest.resetAllMocks();
+
+    jest.spyOn(auroradb, 'getServiceByIdAndDataSource').mockResolvedValue(mockRawService);
+    const s3Spy = jest.spyOn(userData, 'putUserDataInProductsBucketWithFilePath');
+    s3Spy.mockImplementation(() => Promise.resolve('pathToFile'));
+});
+
+describe('returnService', () => {
+    it('should return 302 redirect to /returnService when there is no body in the request', async () => {
```

Comment:
We should be checking that the errors in the session has been updated with the appropriate error message. So the name of the test needs to be 'should return 302 redirect to /returnService with an error when there is no body in the request'